src/libraries/System.Security.Cryptography/tests/KmacXof128Tests.cs

```diff
@@ -0,0 +1,421 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Security.Cryptography.Tests
+{
+    public class KmacXof128Tests : KmacTestDriver<KmacXof128Tests.Traits, KmacXof128>
+    {
+        public class Traits : IKmacTrait<KmacXof128>
+        {
+            public static KmacXof128 Create(byte[] key, byte[] customizationString) =>
+                new KmacXof128(key,customizationString);
```

Comment:
```suggestion
                new KmacXof128(key, customizationString);
```

"Look, I'm adding value!"